src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterface.cs

```diff
@@ -1179,11 +1180,18 @@ public override void WriteDebugLine(string message)
             }
             else
             {
-                // NTRAID#Windows OS Bugs-1061752-2004/12/15-sburns should read a skin setting here...
-                WriteLine(
-                    DebugForegroundColor,
-                    DebugBackgroundColor,
-                    StringUtil.Format(ConsoleHostUserInterfaceStrings.DebugFormatString, message));
+                if (ExperimentalFeature.IsEnabled("PSAnsiRendering"))
+                {
+                    message = Utils.GetFormatStyleString(Utils.FormatStyle.Debug) + Utils.GetOutputString(message, isHost: true, SupportsVirtualTerminal, Console.IsOutputRedirected);
+                    WriteLine(StringUtil.Format(ConsoleHostUserInterfaceStrings.DebugFormatString, message));
```

Comment:
The VT sequences for debug color is added to the message, but `StringUtil.Format(ConsoleHostUserInterfaceStrings.DebugFormatString, message)` will result in `Debug: <message>`, and thus `Debug: ` will have the default color instead of the debug color.

Reply:
It would be nice if we have tests to cover this.

Reply:
yes, will format first and add test